docs/training/model-management/checkpoints.rst

```diff
@@ -323,7 +323,7 @@ The ``--latest`` and ``--uuid`` options are used as follows:
    det trial download <trial_id> --uuid <uuid-for-checkpoint>
 
 Finally, the ``det experiment download`` command provides a similar experience to using the
-:class:`~determined.experimental.client.ExperimentReference` Python API.
+:class:`~determined.experimental.client.ExperimentReference` Python SDK.
```

Comment:
I would recommend removing `Python SDK` here entirely, actually.

> ... provides a similar experience to using the ExperimentReference.

seems fine

Reply:
Or alternately, change the text of the link, like this:

```rst
provides a similar experience to using the
:class:`Python SDK <determined.experimental.client.ExperimentReference>`.
```

Which would render as:

> ... provides a similar experience to using the Python SDK.

Which is more similar to how we link to the Python SDK in these other examples.

Reply:
done